Project.toml

```diff
@@ -2,17 +2,19 @@ name = "CodecBzip2"
 uuid = "523fee87-0ab8-5b00-afb7-3ecf72e48cfd"
 license = "MIT"
 authors = ["Kenta Sato <bicycle1885@gmail.com>"]
-version = "0.6.0"
+version = "0.7.0"
```

Comment:
I'm not sure about this version bump: the API is totally unchanged, but it raises the required Julia version from 1.0 to 1.3, which is weird to do in a point release. Maybe better to bump to 1.0 so that there are both major and minor numbers to work with?